README.md

```diff
@@ -121,6 +150,23 @@ YARN_CONF_DIR=/root/ephemeral-hdfs/conf/ MASTER=yarn-client ./sparkR
 YARN_CONF_DIR=/root/ephemeral-hdfs/conf/ ./sparkR examples/pi.R yarn-client
 ```
 
+## Running on a cluster using sparkR-submit
+
+sparkR-submit is a script introduced to faciliate submission of SparkR jobs to a Spark supported cluster (eg. Standalone, Mesos, YARN).
+It supports the same commandline parameters as [spark-submit](http://spark.apache.org/docs/latest/submitting-applications.html). SPARK_HOME and JAVA_HOME must be defined.
+
+On YARN, YARN_HOME must be defined. Currently, SparkR only supports [yarn-client](http://spark.apache.org/docs/latest/running-on-yarn.html) mode.
```

Comment:
Minor correction: With sparkR-submit its not just yarn-client, but you can also use yarn-cluster for running batch scripts (yarn-cluster doesn't work with the shell).